src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -23,9 +23,10 @@
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.Query.FilterOperator;
 import com.google.appengine.api.datastore.Query.SortDirection;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.UUID;
+//import java.util.ArrayList;
+//import java.util.List;
+//import java.util.UUID;
```

Comment:
If you don't need these lines then I would just get rid of 'em. It's easy enough to bring them back if you need them later.